tests/adapter_daphne/test_daphne.py

```diff
@@ -112,7 +112,12 @@ async def fake_app(*args, **kwargs):
 
 @override_application_settings({"transaction_name.naming_scheme": "framework"})
 def test_daphne_200(port, app):
-    @validate_transaction_metrics(callable_name(app))
+    @validate_transaction_metrics(
```

Comment:
In the instrumentation there are 2 cases for both Daphne and Hypercorn that we need to test. One where there is already an active transaction and one where we start one. I think we should make sure we test both cases here if we can. I'm not sure why it wasn't already tested in here actually.